test/passing/open_preserve.ml.ref

```diff
@@ -182,23 +184,24 @@ let _ =
   let (* b *)
   open (* c *) struct
     type t
-  end
-  (* d *) in
```

Comment:
comment move inside.

Reply:
This is due to the new within/within heuristics, and I didn't manage to fix this without reverting to the state previous to this PR.